docs/contributing-to-airbyte/migrate-connector-to-poetry.md

```diff
@@ -0,0 +1,108 @@
+# Migrate connector to use Poetry and base Docker Image
+
+1. Remove the `Dockerfile` and `.dockerignore`
+2. Include the `baseImage` to the `metadata.yaml` file. For now, look for `baseImage` to find examples:
+```yaml
+data:
+  allowedHosts:
+    hosts:
+      - *
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
+  connectorSubtype: api
+  connectorType: source
+  dockerImageTag: 0.1.0
+```
+3. Create a file called `pyproject.toml` and transfer all lib used in `setup.py` to this file
```

Comment:
Users could also try `airbyte-ci --name=<connector-name> migrate_to_poetry`
It automatically does what you described below.